final/pkg/config/config.go

```diff
@@ -0,0 +1,22 @@
+package config
+
+import (
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/mysql"
+)
+
+var (
+	db *gorm.DB
```

Comment:
We should avoid using global variables. This can be solved using dependency injection. We can inject db instance to the handlers directly.